styles/globals.css

```diff
@@ -0,0 +1,69 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+:root {
+--background: #020202;
+--primary:#0d2818;
+--accent:#415a77;
+--light:#e0e1dd;
+--dark:#0e1128;
+}
+
+@media (prefers-color-scheme: dark) {
+  :root {
+    --foreground-rgb: 255, 255, 255;
+    --background-start-rgb: 0, 0, 0;
+    --background-end-rgb: 0, 0, 0;
+  }
+}
+
+
+body{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: system-ui, sana serif;
+  background-color:var(--background);
+  color:var(--light);
+}
+
+.postContainer{
+    display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 1rem 0;
+
+}
+.postWrapper{
+    background-color: var(--dark);
+  width: min(700px, (100% - 2rem));
```

Comment:
Can you explain what's happening here? Wouldn't you need the `calc` function to achieve the arithmetic here?

Reply:
Will resolve